lib/portlayer/exec/container.go

```diff
@@ -258,6 +272,11 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 
 		c.State = *h.State
 		commitEvent = events.ContainerStarted
+
+		// refresh the struct with what propery collector provides
+		if err := c.Refresh(); err != nil {
```

Comment:
The refresh happens after reconfigure. Does that mean if there is inconsistency between portlayer and VC, the operation will always fail?
Currently, the VM power on will change the changeVersion, so out of band operation might cause failure with it.